drivers/spi/spi_ll_stm32.c

```diff
@@ -193,7 +193,9 @@ static void spi_stm32_complete(struct spi_stm32_data *data, SPI_TypeDef *spi,
 
 	if (LL_SPI_GetMode(spi) == LL_SPI_MODE_MASTER) {
 		while (LL_SPI_IsActiveFlag_BSY(spi)) {
-			/* NOP */
+			if (LL_SPI_IsActiveFlag_MODF(spi)) {
```

Comment:
Please add a comment directly in the code

Reply:
Done, thank you !